src/main.js

```diff
@@ -0,0 +1,23 @@
+function convertItemsToUnorderedList(listItems) {
+  const ulList = document.createElement("ul");
+  listItems.map(a => {
```

Comment:
We will have to change things here, as there is a bad practice. What happens is that there is a side-effect inside the `map` function argument; you are appending elements to `ulList`. When using `map` we better just return values and have no side effects:

```javascript
const liElements = listItems.map(item => {
  const liElement = document.createElement("li");
  liElement.textContent = item.name;
  return liElement;
});
liElements.forEach(liElement => ulElement.appendChild(liElement));
```

Alternatively, you can use [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) which is semantically better for such use cases;

```javascript
listItems.forEach(item => {
  const liElement = document.createElement("li");
  liElement.textContent = item.name;
  ulElement.appendChild(liElement)
});
```

Up to you!